testing/e2e/e2e_test.go

```diff
@@ -98,7 +98,7 @@ var _ = Describe("kube-vip broadcast neighbor", func() {
 
 			manifestPath := filepath.Join(tempDirPath, "kube-vip-ipv4.yaml")
 
-			for i := 0; i < 2; i++ {
+			for i := 0; i < 3; i++ {
```

Comment:
Maybe a bit slower but using 3 nodes removes the flakiness since etcd won't lose quorum by deleting only one node.